data-prepper-plugins/http-sink/build.gradle

```diff
@@ -26,7 +26,7 @@ dependencies {
     implementation 'com.github.scribejava:scribejava-core:8.3.3'
     implementation project(path: ':data-prepper-core')
     implementation project(':data-prepper-plugins:parse-json-processor')
-    implementation 'com.amazonaws:aws-java-sdk-sts:1.12.395'
+    implementation 'software.amazon.awssdk:sts:2.21.23'
```

Comment:
The BOM defines the version. So you can change this to:

```
implementation 'software.amazon.awssdk:sts'
```

Reply:
removed particular version and updated